### ISSUE TITLE
i3fyra is broken after your update today

### ISSUE THREAD

**Issue by yuceltoluyag** [budlabs/i3ass#170]
```
i3fyra -a
[ERROR] @$ not a valid command
```

https://aur.archlinux.org/packages/i3ass

```
i3fyra --version
i3fyra - version: 1.25
updated: 2022-05-21 by budRich

```

**Comment by budRich** [comment]
Big thanks for reporting this, it was a quite serious bug, and very easy to fix.
Also your snippet with output of `i3fyra --version` revealed another bug, the update date was wrong (it should had been 2022-05-20), apparently the installation date was used, that is also fixed now.

**Comment by yuceltoluyag** [comment]
@budRich Thank you for making such an application for us, again I have detected a few problems. Again, I went back to the previous two versions and tested if the problem was really caused by this version. I do not encounter any errors in this version. Again, I would like other users to test the latest update, maybe the problems are specific to me https://github.com/budlabs/i3ass/releases/tag/2022.05.20.1

After the last patch, Thunar really didn't seem to know where to position. when i try to open it gives this error

```
ERROR: Expected one of these tokens: <word>
ERROR: Your command: [con_id=] focus;
ERROR:                       ^^^^^^^^
```

When I run it by typing "thunar" in the terminal, "thunar" was not focused. (hide, bring forward, i3run features do not work)
